New version: ThreadSafeDicts v0.1.5

JuliaRegistrator opened the pull request:
- Registering package: ThreadSafeDicts
- Repository: https://github.com/wherrera10/ThreadSafeDicts.jl
- Created by: @wherrera10
- Version: v0.1.5
- Commit: c1f209107db05efea485b53604d11ae7ce225f49
- Reviewed by: @wherrera10
- Reference: https://github.com/wherrera10/ThreadSafeDicts.jl/commit/c1f209107db05efea485b53604d11ae7ce225f49#commitcomment-139081646
- Description: Thread safe Julia Dict
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d6220da33723dade6a603585cd1b4150b1bbcafeacd9d31f36bfe595bd2ec549dbbff9abde27a8fbb6ea4a6e8b7f96a01ab7c22c5610f42d82b8883802ed0f3c28764db523025fb6edf69373b550fa88fce40f4556cce4fa2e3246e23667e84a9ccbd252462a25fb6e14d365eaabdb555ede67feb4d9f55092c2fdefbdd56d308b31354e7f3caaa5901126fb8380b9924e39ca7fd4c4d3e019eb2a47e40bf17371c2d4f30e457dc28eead3341c5aa929 -->

DilumAluthge commented:
[noblock]

Looks like there is a merge conflict.

@wherrera10 Can you re-trigger Registrator? This will fix the merge conflict.

JuliaRegistrator committed:
New version: ThreadSafeDicts v0.1.5

UUID: 4239201d-c60e-5e0a-9702-85d713665ba7
Repo: https://github.com/wherrera10/ThreadSafeDicts.jl.git
Tree: 561eecf8f487e7eb66246a8278e7f09205ff8986

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
